.gitignore

```diff
@@ -16,6 +16,10 @@
 # Files created during tests
 /test/com/facebook/buck/testutil/integration/testlibs/buck-out/
 
+# IntelliJ
+*.iml
```

Comment:
I think buck purposefully checks in `.iml` files and perhaps things in `.idea`

Reply:
this will ignore changes to any .idea files in the future. I don't believe they check in iml files